tools/devops-tasks/GitHubPRComment/readme.md

```diff
@@ -0,0 +1,39 @@
+# GitHub PR Comment Task
+
+This task has the functionality to receive plain text files and use the content to create a GitHub comment on a Pull Request.
+
+## Configuration
+
+### User Token
+
+[GitHub Personal Access Token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) (it must have *Repo* permissions).
+
+### Repository
+
+RepositoryOwner/Repository name. Use  `$(Build.Repository.Name)` if you want to run on the build's repository.
+
+### Pull Request Number
+
+Number of the pull request you want to comment. Use `$(System.PullRequest.PullRequestNumber)` if you want to run on the build's PR.
+
+### Path to the file/s
+
+Path to the file/s that you want to get the PR comments from. If you specify a folder, it will pick all the files that have the specified extension inside that folder
```

Comment:
```suggestion
Path to the file/s that you want to get the PR comments from. If you specify a folder, it will pick all the files that have the specified extension inside that folder.
```